exporter/exporter_test.go

```diff
@@ -100,3 +105,30 @@ EOT`,
 		})
 	}
 }
+
+func TestRunExport(t *testing.T) {
+	var server *httptest.Server
+	server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		assert.Equal(t, "/public/v0.2/blars-org/projects/blars/metric_dashboards/123abc", r.URL.Path)
+		body, err := json.Marshal(map[string]client.UnifiedDashboard{
+			"data": {
+				ID:   "123abc",
+				Type: "metric_dashboard",
+				Attributes: client.UnifiedDashboardAttributes{
+					Name:   "blars dashboard",
+					Charts: []client.UnifiedChart{},
+				},
+			},
+		})
+		assert.NoError(t, err)
+		_, err = w.Write(body)
+		assert.NoError(t, err)
+		w.WriteHeader(http.StatusOK)
+	}))
+	defer server.Close()
+
+	os.Setenv("LIGHTSTEP_API_KEY", "api-key")
+	os.Setenv("LIGHTSTEP_ORG", "blars-org")
+	os.Setenv("LIGHTSTEP_API_URL", server.URL)
+	Run("exporter", "export", "lightstep_dashboard", "blars", "123abc")
```

Comment:
Thanks for adding this! Just two things I would add here:

1. An assertion for `nil` error return value from `Run`.
2. Communication between the harness and the HTTP handler so you know its your handler that's exercised and not some other handler in the event of a healthy result from `Run`. This is a synchronous action so you could use an atomic for that, but you could use something like this, too:

```go
done := make(chan struct{})

server = httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
    defer close(done)
    // Remainder of HTTP handler...
}))

// All environment setting and Run call

select {
case <-time.After(10 * time.Second):
    t.Fatal("timeout waiting for HTTP handler to be exercised")
case <-done:
}

```